modules/installation-network-user-infra.adoc

```diff
@@ -255,6 +255,11 @@ ifdef::vsphere[]
 :!vsphere:
 endif::[]
 
+[discrete]
+== NTP configuration
+
+An {product-title} cluster that is installed in a restricted network is configured to use the public NTP server `rhel.pool.ntp.org` by default. To avoid clock skew, reconfigure the cluster to use a private NTP server instead. See the documentation for _Configuring chrony time service_ for more information.
```

Comment:
I don't think there is much value in calling out specifically which NTP server is used by default. In this case, the user does not care about that detail, since they need to change it anyway. Including the detail means that it is something that could get out-of-date in the future.
```suggestion
An {product-title} cluster that is installed in a restricted network is configured to use a public NTP server by default. To avoid clock skew, reconfigure the cluster to use a private NTP server instead. See the documentation for _Configuring chrony time service_ for more information.
```